apps/src/lib/util/urlHelpers.js

```diff
@@ -2,21 +2,28 @@ import $ from 'jquery';
 import _ from 'lodash';
 
 /**
- * Attempt to construct an absolute Pegasus url (that is,
+ * Attempt to construct an absolute Marketing url (that is,
  * starting with https://code.org or the appropriate
  * equivalent for the current environment) from a given
- * relative url.  If we're already on pegasus we'll
+ * relative url. If we're already on marketing we'll
  * just return the relative url.
  * @param {string} relativeUrl - should start with a
  *   leading slash.
  */
-export function pegasus(relativeUrl) {
+export function marketingUrl(relativeUrl) {
```

Comment:
We're calling this the `corporateWebsite` now (the code.org marketing site)

```suggestion
export function corporateWebsiteUrl(relativeUrl) {
```